components/InteriorLeftNavItem.js

```diff
@@ -8,8 +8,8 @@ const propTypes = {
   activeHref: PropTypes.string,
   tabIndex: PropTypes.number,
   onClick: PropTypes.func,
-  label: PropTypes.string.isRequired,
   blankTarget: PropTypes.bool,
+  children: PropTypes.node,
```

Comment:
Linting doesn't seem to be complaining about `children` missing from `defaultProps`. How do I even do that?